README.md

```diff
@@ -22,6 +22,19 @@ config :movie_wager_api, MovieWagerApi.Repo,
 Similarly, you will want to follow the same steps for `test.secret.example.exs`.
 The secret files are not checked into source control, so don't worry about your credentials being shared.
 
+## Setting up Twitter Authentication
+
+The `dev.secret.exs` and `test.secret.exs` files also include information for the twitter OAuth API.
```

Comment:
We have an secret.example.exs file, correct? We should add in the comments to look to that file if there are questions.
We should also make sure that file is up to date with everything that needs to be specified and enough comments for explanations.